### ISSUE TITLE
a11y: Color Contrast - Cluster Appearance

### ISSUE THREAD

**Issue by rak-phillip** [rancher/dashboard#13742]
Insufficient color contrast for standard text

Insufficient color contrast for the border of form control

## Actual Result

The mentioned form control border have insufficient contrast with their background, making it difficult for people with low vision and color blindness to distinguish them.

The standard text with background/foreground color combinations having an insufficient contrast ratio making it difficult for people with low vision and color blindness to read standard text elements.

## Instances

"Foreground color: #DCDEE7
Background color: #FFFFFF
Contrast ratio: 1.34:1
Main Content
- input field associated with transparent text (which gets available after selecting Badge background color) "

"Foreground Text Color: #B6B6C2
Background Text Color: #FFFFFF
Ratio: 2.00:1 
Main Content
- transparent (becomes available when checkbox is selected) 

Foreground Text Color: #FFFFFF
Background Text Color: #297DB4
Ratio: 4.48:1 
Main Content
- Cancel and Apply Button (on focus)

Foreground Text Color: #3D98D3
Background Text Color: #FFFFFF
Ratio: 3.16:1 
Main Content
- Icon Text and Custom comment (in typing state, available when you select the checkbox)
- Cancel Button (in default state)

Foreground Text Color: #FFFFFF
Background Text Color: #3D98D3
Ratio: 3.16:1 
Main Content
- Apply Button (in default state)
"

## Expected result

The contrast ratio between the foreground border color of the form field and background color should be sufficient.

The contrast ratio between foreground text and background for the standard text should be sufficient.

**Comment by rak-phillip** [comment]
Issue 140 fails VPAT. No notes. Original comment:

> The mentioned form control border have insufficient contrast with their background, making it difficult for people with low vision and color blindness to distinguish them.

Issue 141 fails VPAT:

> "Actual Result: The standard text with background/foreground color combinations having an insufficient contrast ratio making it difficult for people with low vision and color blindness to read standard text elements.
> 
> Instances:
> Foreground Text Color: #3B71E1
> Background Text Color: #EDF2FC
> Ratio: 4.4:1 
> Main Content
> - Cancel button (on focus and hover)
> 
> Recommendation: On all pages, ensure that each foreground/background color combination presents a contrast ratio of at least 4.5:1 for standard text."
